packages/strapi-plugin-upload/admin/src/components/BrowseAssets/index.js

```diff
@@ -67,6 +68,23 @@ const BrowseAssets = () => {
     });
   };
 
+  /* eslint-disable indent */
+  /* eslint-disable react/jsx-indent */
+  const renderCardControl = noNavigation
+    ? null
+    : id => (
+        <CardControl
+          small
+          title="edit"
+          color="#9EA7B8"
+          type="pencil"
+          onClick={() => handleGoToEditFile(id)}
+        />
+      );
```

Comment:
This is a bit strange. Can't we do something like 
```js
const renderCardControl = id => {
    if (noNavigation) {
        return null
    }
    
    return (
        <CardControl
          small
          title="edit"
          color="#9EA7B8"
          type="pencil"
          onClick={() => handleGoToEditFile(id)}
        />
      );
}
```

Reply:
Well if you prefer but it is equivalent to not passing a fender méthode so personally I'd rather have no method called if it's not needed